src/pages/docs/guides/chat/build-embedded-chat.mdx

```diff
@@ -0,0 +1,414 @@
+* **Deliveries:** Know if the delivery driver or customer is actively monitoring the chat.
+* **Gaming sessions:** See which players are currently active in the game.
+* **Collaborative work:** Know who's available for real-time discussion.
+
+Beyond just online/offline status, presence can include rich information:
+
+* **Current status:** "Available", "In a meeting", "Driving", "On break".
+* **Task context:** "Working on ticket #123", "In delivery zone A".
+* **User info:** Use presence to show an avatar, display name, role, or other information about the user.
+
+This contextual presence information helps participants understand not just who's online, but who's ready and able to engage with the current task.
+
+```javascript
+// subscribe to presence events
+const { unsubscribe } = room.presence.subscribe((event) => {
+  console.log(`${event.member.clientId} entered with data: ${event.member.data}`);
+
+  const presentMembers = await room.presence.get();
+  console.log('Present members:', presentMembers);
+});
+
+// attach the room
+await room.attach();
+
+// join presence
+await room.presence.enter({ status: 'available' });
+
+// change presence data
+await room.presence.update({ status: 'busy' });
+
+// leave presence
+await room.presence.leave();
+```
+
+## Typing indicators
+
+[Typing indicators](/docs/chat/rooms/typing) are a common feature in most chat applications. They show when someone is actively composing a message, helping to:
+
+* **Manage expectations:** Users know when a response is being prepared.
+* **Reduce duplicate messages:** See that someone is already addressing the question.
+* **Improve flow:** Better conversation pacing in support and collaborative scenarios.
+
+In Ably Chat typing indicators are a core feature with a simple API:
+
+```javascript
+const room = await chatClient.rooms.get('support-ticket-123');
+
+// show who's typing
+room.typing.subscribe((event) => {
+  console.log('Currently typing:', event.currentlyTyping);
+});
+
+// attach the room
+await room.attach();
+
+// show a typing indicator (call on every keystroke, Ably Chat SDKs throttle this to a predefined interval)
+await room.typing.keystroke();
+
+// stop the typing indicator (for example when a message is sent)
+await room.typing.stop();
+```
+
+## Message reactions
+
+Message reactions are a ubiquitous feature in modern chat experiences and are a great way to enhance engagement and enable users to quickly express sentiment to key points in the conversation.
+
+Send a message reaction:
+<Code>
+```javascript
+const message; // your message
+await room.messages.reactions.send(message.serial, {name: '❤️'});
+```
+</Code>
+
+Message reactions in Ably Chat come in three types: `unique`, `distinct` and `multiple`, to suit different use cases: from one reaction per message to multiple reactions per message with or without counts. See the [Message reactions](/docs/chat/rooms/message-reactions) documentation for more details.
```

Comment:
Which is most important to this use case? We shouldn't just be re-listing our features here as someone can find that from the conceptual docs. This is about us being opinionated on what's important to this use case.

Reply:
I can see how this is too much detail.

But there's no answer about which to recommend. It's really application-dependent.

Maybe just remove the paragraph?